### ISSUE TITLE
Error setting up merchant feature on 3.0.1

### ISSUE THREAD

**Issue by nkgilley** [spesmilo/electrum#3262]
I'm trying to setup electrum payments on my website via the merchant feature on 3.0.1 but I'm getting an error.  This works fine for me on 2.9.3.

I used letsencrypt to generate my certificates, and configured the ssl_privkey to privkey.pem and ssl_chain to fullchain.pem. Next I setup apache, start the electrum daemon, load_wallet, and perform an addrequest which comes back with this error:

```bash
$ electrum addrequest 3.14 -m "this is a test"
Invalid call parameters: <Fault -32602: Invalid parameters: 'bytearray(b\'0\\x82\\x05+0\\x82\\x04\\x13\\xa0\\x03\\x02\\x01\\x02\\x02\\x12\\x03c\\xc3\\xcb\\x80\\ has type str, but expected one of: bytes>
Traceback (most recent call last):
  File "/home/wagershark/virtualenvs/electrum/bin/electrum", line 415, in <module>
    result = server.run_cmdline(config_options)
  File "/home/wagershark/virtualenvs/electrum/lib/python3.5/site-packages/jsonrpclib/jsonrpc.py", line 650, in __call__
    return self.__send(self.__name, args)
  File "/home/wagershark/virtualenvs/electrum/lib/python3.5/site-packages/jsonrpclib/jsonrpc.py", line 532, in _request
    check_for_errors(response)
  File "/home/wagershark/virtualenvs/electrum/lib/python3.5/site-packages/jsonrpclib/jsonrpc.py", line 1220, in check_for_errors
    raise ProtocolError((code, message))
jsonrpclib.jsonrpc.ProtocolError: (-32602, "Invalid parameters: 'bytearray(b\\'0\\\\x82\\\\x05+0\\\\x82\\\\x04\\\\x13\\\\xa0\\\\x03\\\\x02\\\\x01\\\\x02\\\\x02\\\\x12\\\\x03c\\\\xc3\\\\xcb\\\\x80\\\\ has type str, but expected one of: bytes")
```

**Comment by ariard** [comment]
I think it's jsonrpclib package is the cause of the disorder, is your package jsonrpclib-pelix ? The 3.0.1 is in python3 and must use this one.

**Comment by nkgilley** [comment]
$ pip list | grep jsonrpc
jsonrpclib-pelix (0.3.1)

**Comment by lzsaver** [comment]
@SomberNight, can you have a look?

**Comment by ecdsa** [comment]
yes it is related to RSA signing. I can reproduce it

**Comment by ecdsa** [comment]
fixed in 60e0b43b8129fe1b66a6ceeb9f2dc01a5b6de1b4
